lib/screens/information_screen.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:kledd/lang/my_localizations.dart';
+import '../widgets/information_segment.dart';
+import 'package:smooth_page_indicator/smooth_page_indicator.dart';
+
+class InformationScreen extends StatefulWidget {
+  @override
+  _InformationScreenState createState() => _InformationScreenState();
+}
+
+class _InformationScreenState extends State<InformationScreen> {
+  final controller = PageController(viewportFraction: 0.8);
+
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Theme.of(context).backgroundColor,
+      body: Column(
+        mainAxisAlignment: MainAxisAlignment.spaceAround,
+        children: [
+          Center(
+            child: Directionality(
+              textDirection: TextDirection.ltr,
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: <Widget>[
+                  SizedBox(height: 16),
+                  SizedBox(
+                    height: 500,
+                    child: PageView(
+                      controller: controller,
+                      children: List.generate(
+                        3,
+                        (index) => InformationSegment(info: MyLocalizations.of(context).howToText(index)),
+                      ),
+                    ),
+                  ),
+                  SizedBox(height: 16),
+                  SmoothPageIndicator(
+                    controller: controller,
+                    count: 3,
+                    effect: JumpingDotEffect(
+                      dotColor: Colors.blueGrey,
+                      activeDotColor: Theme.of(context).accentColor,
+                      dotHeight: 8,
+                      dotWidth: 8,
+                    ),
+                    onDotClicked: (index) {
+                      // TODO: Place function that changes Card here?
```

Comment:
Maybe just skip this if we do not want this behaviour or open an issue to implement it, it is up to you. If this function is required just replace it with null. The important thing is to remove the TODO if we are not sure that this should be done. This will prevent us from ending up with lots of TODOs in the final product.